<launch>
    <node pkg="billy_camarm" type="billy_camarm_sample.py" name="billy_camarm_node" output="screen">
    </node>

    <!--node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
        <remap from="image" to="/image_processed" />
    </node-->
</launch>
